vavr/src/main/java/org/jdbi/v3/vavr/VavrTupleRowMapperFactory.java

```diff
@@ -111,24 +129,31 @@ private Tuple buildTuple(Class<? extends Tuple> tupleClass, MapperValueResolver
                 .map(cm -> new SingleColumnMapper<>(cm, colIndex));
     }
 
-    Optional<RowMapper<?>> getRowMapper(Type type, int tupleIndex, ConfigRegistry config) {
+    private Optional<RowMapper<?>> getRowMapper(Type type, int tupleIndex, ConfigRegistry config) {
 
         // try to map by configured tuple
-        Optional<String> tupleColumn = Optional.ofNullable(config.get(TupleMappers.class)
-                .getColumn(tupleIndex));
+        Option<String> tupleColumn = getConfiguredColumnName(tupleIndex, config);
 
-        if (tupleColumn.isPresent()) {
+        if (tupleColumn.isDefined()) {
```

Comment:
With the changes in the main map method, I believe we can get rid of this if block and the `tupleColumn` variable now.

Reply:
good catch